src/1776B0.c

```diff
@@ -134,7 +601,740 @@ ApiStatus func_8024CE9C(Evt* script, s32 isInitialCall) {
+                D_8029F284 = 8;
+                D_8029F29E = 120;
+                D_8029F286 = 0x10;
+                D_8029F2A2 = 1;
+                D_8029F2A3 = 1;
+                D_8029F2A5 = 1;
+                preset = &CamPreset_I;
+                D_8029F282 = 0;
+                D_8029F2A6 = 0;
+                D_8029F27C = 256;
+                break;
+            case 0x32:
+                D_8029F280 = 320;
+                D_8029F284 = 8;
+                D_8029F29E = 5;
+                D_8029F2A2 = 1;
+                D_8029F282 = 0;
+                D_8029F286 = 0;
+                D_8029F2A3 = 0;
+                preset = &CamPreset_M;
+                D_8029F27C = 256;
+                D_8029F2A6 = 0;
+                break;
+            case 0x1D:
+            case 0x33:
+                D_8029F29E = 50;
+                D_8029F280 = 500;
+                D_8029F286 = 0;
+                preset = &CamPreset_D;
+                break;
+            case 0x34:
+                D_8029F280 = 280;
+                D_8029F284 = 8;
+                D_8029F29E = 30;
+                D_8029F286 = -4;
+                D_8029F2A2 = 1;
+                D_8029F282 = 0;
+                D_8029F2A3 = 0;
+                preset = &CamPreset_M;
+                D_8029F2A6 = 0;
+                D_8029F27C = 256;
+                break;
+            case 0x35:
+                D_8029F280 = 380;
+                D_8029F284 = 8;
+                D_8029F29E = 60;
+                D_8029F2A2 = 1;
+                D_8029F2A5 = 1;
+                D_8029F282 = 0;
+                D_8029F286 = 0;
+                D_8029F2A3 = 0;
+                preset = &CamPreset_M;
+                D_8029F2A6 = 0;
+                D_8029F27C = 256;
+                break;
+            case 0x36:
+                D_8029F280 = 220;
+                D_8029F284 = 8;
+                D_8029F29E = 30;
+                D_8029F286 = 24;
+                D_8029F2A2 = 1;
+                D_8029F2A3 = 1;
+                preset = &CamPreset_I;
+                D_8029F282 = 0;
+                D_8029F2A6 = 0;
+                D_8029F27C = 256;
+                break;
+            case 0x37:
+                D_8029F280 = 210;
+                D_8029F284 = 8;
+                D_8029F29E = 20;
+                preset = &CamPreset_N;
+                D_8029F282 = 0;
+                D_8029F286 = 0xA;
+                D_8029F2A2 = 0;
+                D_8029F2A3 = 0;
+                D_8029F278 = 0.0f;
+                D_8029F270 = -95.0f;
+                D_8029F274 = 22.0f;
+                break;
+            case 0x38:
+                D_8029F280 = 320;
+                D_8029F284 = 8;
+                D_8029F29E = 30;
+                D_8029F286 = -4;
+                D_8029F2A2 = 1;
+                D_8029F282 = 0;
+                D_8029F2A3 = 0;
+                D_8029F27C = 256;
+                preset = &CamPreset_M;
+                break;
+            case 0x39:
```

Comment:
So this is why this page was laggy.

I think we should introduce an enum for cam presets describing what the camera looks at or is used for (probably have the name be the usage situation, with a comment describing it e.g.`BTLCAM_PRESET_TATTLE // Really zoomed in`). Clover may have already named these - it's worth having a look.

Reply:
fun fact- at the time of decomp, this was the game's 14th largest un-decompiled function